packages/babel-plugin/src/babel-plugin.tsx

```diff
@@ -85,6 +86,8 @@ export default declare<State>((api) => {
             opts: { importReact: shouldImportReact = true },
           } = state;
 
+          preserveFlowComment(path);
```

Comment:
Should we make this flow agnostic? Maybe `preserveLeadingComment`?

Reply:
Yep, the intention for the original ticket was for the solution to be agnostic. Flow was just a direct use-case. You should either change the logic to insert everything after the original `@compiled/react` import before it's removed or insert it first, then move all the following leading comments back up

Reply:
@MonicaOlejniczak Just to clarify with everything do you mean the existing imports or comments?